x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/engine_creation.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiForm,
+  EuiFlexGroup,
+  EuiFormRow,
+  EuiFlexItem,
+  EuiFieldText,
+  EuiSelect,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiButton,
+  EuiPanel,
+} from '@elastic/eui';
+import { useActions, useValues } from 'kea';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { FlashMessages } from '../../../shared/flash_messages';
+import {
+  ALLOWED_CHARS_NOTE,
+  ENGINE_CREATION_FORM_ENGINE_LANGUAGE_LABEL,
+  ENGINE_CREATION_FORM_ENGINE_NAME_LABEL,
+  ENGINE_CREATION_FORM_ENGINE_NAME_PLACEHOLDER,
+  ENGINE_CREATION_FORM_SUBMIT_BUTTON_LABEL,
+  ENGINE_CREATION_FORM_TITLE,
+  ENGINE_CREATION_TITLE,
+  SANITIZED_NAME_NOTE,
+  SUPPORTED_LANGUAGES,
+} from './constants';
+import { EngineCreationLogic } from './engine_creation_logic';
+
+export const EngineCreation: React.FC = () => {
+  const { name, rawName, language } = useValues(EngineCreationLogic);
+  const { setLanguage, setRawName, submitEngine } = useActions(EngineCreationLogic);
+
+  return (
+    <div data-test-subj="EngineCreation">
+      <SetPageChrome trail={[ENGINE_CREATION_TITLE]} />
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>{ENGINE_CREATION_TITLE}</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageBody>
+        <FlashMessages />
+        <EuiPanel>
```

Comment:
[Super optional/not a change request, just me thinking out loud] I think if we use `EuiPageBody` here that shows up w/ the panel appearance as well - that being said I don't 100% know what the semantic distinction is over PageBody vs Panel, or how much that's changing in Amsterdam 🤔 Mostly just posting to see if anyone else does haha

Reply:
Thanks for clarifying, I'll keep this in mind as I'm wrapping up Meta Engine Creation